testing/e2e/app/routes/ComboBox/spec.ts

```diff
@@ -1,181 +1,382 @@
-import { test, expect } from '@playwright/test';
+import { test, expect, Page } from '@playwright/test';
+
+const defaultOptions = [
+  { label: 'Item 0', value: 0 },
+  { label: 'Item 1', value: 1, subLabel: 'sub label' },
+  { label: 'Item 2', value: 2 },
+  { label: 'Item 3', value: 3 },
+  { label: 'Item 4', value: 4 },
+  { label: 'Item 10', value: 10 },
+  { label: 'Item 11', value: 11 },
+];
 
 test.describe('ComboBox', () => {
-  test('should support keyboard navigation when virtualization is enabled', async ({
-    page,
-  }) => {
-    await page.goto('/ComboBox?virtualization=true');
-
-    await page.keyboard.press('Tab');
-    const comboBoxInput = page.locator('#test-component').locator('input');
-    const comboBoxMenu = page.getByRole('listbox');
-    await expect(comboBoxInput).toHaveAttribute(
-      'aria-controls',
-      'test-component-list',
-    );
-    await expect(comboBoxMenu).toBeVisible();
+  test.describe('General', () => {
```

Comment:
nit: do not nest `describe` blocks; instead create sibling `describe` blocks at the top-level. The existing `describe` block could be renamed to something like "ComboBox (general)" if you want.

same comment applies to other tests that use this pattern